foreign_cc/repositories.bzl

```diff
@@ -13,7 +13,8 @@ def rules_foreign_cc_dependencies(
         make_version = "4.3",
         ninja_version = "1.10.2",
         register_preinstalled_tools = True,
-        register_built_tools = True):
+        register_built_tools = True,
+        register_toolchains = True):
```

Comment:
Why does registering the toolchains need to be optional for bzlmod support?

Reply:
bzlmod removes the register_toolchains call and has its own mechanism for registering toolchains in the `MODULE.bazel` file. I haven't fully understood how we can make it configurable in the same way we currently do with `bzlmod`.